Source/WebCore/platform/graphics/Font.h

```diff
@@ -52,6 +52,12 @@
 #include <usp10.h>
 #endif
 
+#if USE(SKIA)
+WTF_IGNORE_WARNINGS_IN_THIRD_PARTY_CODE_BEGIN
+#include <skia/core/SkTextBlob.h>
```

Comment:
We shouldn't pull in Skia headers here if avoidable -- since SkTextBlob isn't stored as member, a `class SkTextBlob` should be sufficient, no?